app/utils/constants.ts

```diff
@@ -147,6 +147,27 @@ const PROVIDER_LIST: ProviderInfo[] = [
     getApiKeyLink: 'https://lmstudio.ai/',
     labelForGetApiKey: 'Get LMStudio',
     icon: "i-ph:cloud-arrow-down",
+  },
+  {
+    name: 'NovitaAI',
+    staticModels: [
+      { name: 'qwen/qwen-2.5-72b-instruct', label: 'qwen-2.5-72b-instruct', provider: 'NovitaAI', maxTokenAllowed: 16000 },
+      { name: 'meta-llama/llama-3.1-405b-instruct', label: 'llama-3.1-405b-instruct', provider: 'NovitaAI', maxTokenAllowed: 16384 }
+    ],
+    getApiKeyLink: 'https://novita.ai/settings/api-keys'
```

Comment:
I seen you comment, can you leave commented out model list url here, I do plan at some point to refactor so that all model lists go trough our backend where cors will not be a problem.

Reply:
https://api.novita.ai/v3/openai/models 

this is api to get list of models. but cors not allowed to need to handle thru backend proxy.